### PR TITLE
Feature/form element textarea

### PR DESCRIPTION
**Pull request by ronaldo-mh**

### CODE REVIEW

**Comment by kikoruiz** [comment]
Hello, I just have a doubt: Are these components the same defined in https://paper.dropbox.com/doc/SUI-Form-Elements-WIP-dLZacE1Ye7Qxrp7n9Y0Hy?

**Comment by ronaldo-mh** [comment]
@kikoruiz No, initially they are not the same defined in the dropbox paper for SUI form elements.
The design delivered by our team's UX designer has some differences, but in the moment he created it (have into account that this was designed thinking in the New Constructions' detail page contact form), he validated it with the SUI UX team, even though in that moment there was no specifications for the form elements.

Having said that, I see that the changes required to be fully compliant with the specification (it is only styles differences) are neither extensive nor complicated.

¿Do you think it would be better if I adapt this styles now (taking into account that the definition of form elements is still in progress, and that I don't have the sketch/zeplin, so I don't know the margins/paddings/colours/font-sizes/borders/etc.)?
Or could I deliver this and then adapt it when the form elements specification goes from WIP to DONE?

File: components/form/textarea/src/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+   */
+  cols: PropTypes.number,
+  /**
+   * Title label to show above the text area element.
+   */
+  label: PropTypes.string,
+  /**
+   * Specifies a name for the text area.
+   */
+  name: PropTypes.string.isRequired,
```
Comment (jcger) [inline components/form/textarea/src/index.js]: Why is this required?

**Review by jcger — APPROVED** [review]

File: components/form/textInput/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### FormTextInput
```
Comment (jcger) [inline components/form/textInput/README.md]: Please check the FormTextInput files here, they might not be part of this PR

**Comment by kikoruiz** [comment]
@frmedina Well, iterating over this component sounds the best option, but let other SUI-developers know that this component is not finished yet. README can be a good place for it.

**Review by kikoruiz — APPROVED** [review]

File: components/form/textarea/src/index.js (hunk 1 above)
Comment (ronaldo-mh) [reply components/form/textarea/src/index.js]: Same as pointed in the checkbox's PR, I will remove this 👍

File: components/form/textInput/README.md (hunk 2 above)
Comment (ronaldo-mh) [reply components/form/textInput/README.md]: Yes, my mistake, pushed to wrong branch.
I'll fix it now, thanks! 👍

**Comment by ronaldo-mh** [comment]
@kikoruiz Great idea, I will write something in the README to clarify it!
